.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "extends": [
```

Comment:
I would add following plugins to that configuration:

- simple-import-sort
- import
- react-hooks
- unused-imports

Also I would add some rules:

`'react-hooks/exhaustive-deps': 'warn',
    'react-hooks/rules-of-hooks': 'error',
'no-unexpected-multiline': 'error',
'block-spacing': ['error', 'always'],
    'no-whitespace-before-property': 'error',
    'space-before-blocks': ['error', 'always'],
        'unused-imports/no-unused-imports': 'error',
    'unused-imports/no-unused-vars': [
      'error',
      {
        vars: 'all',
        varsIgnorePattern: '^_',
        args: 'after-used',
        argsIgnorePattern: '^_',
      },
    ],
    'import/no-cycle': [
      'warn',
      {
        ignoreExternal: true,
      },
    ],
    'simple-import-sort/imports': [
      'error',
      {
        groups: [
          ['^\\u0000'],
          ['^@?\\w'],
          [`^(${folders.join('|')})(/.*|$)`, '^\\.'],
          ['^\\.'],
        ],
      },
    ],
    'simple-import-sort/exports': 'error',
    'react/jsx-uses-react': 'off',
    'react/react-in-jsx-scope': 'off',
    'curly': 'error',    
`

Reply:
Use three backticks